get_genotype_vcf.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error
+
+"""
+This script will
+
+- extract common variants
+- export as VCF (one per chrom)
+
+this will be used as input for the
+SAIGE-QTL association pipeline.
+"""
+
+# python modules
+from cpg_utils.hail_batch import (
+    dataset_path,
+    get_batch,
+    get_config,
+    init_batch,
+    output_path,
+)
+
+import click
+import hail as hl
+from hail.methods import export_vcf
+
+
+BIOHEART_TOB_VDS = dataset_path('vds/1-0.vds')
+BCFTOOLS_IMAGE = get_config()['images']['bcftools']
+
+# inputs:
+@click.option('--chromosomes', help=' eg chr22')
+@click.command()
+def main(chromosomes):
+    """
+    Run associaTR processing pipeline
+    """
+
+    init_batch()
+
+    vds_path = BIOHEART_TOB_VDS
+    vds = hl.vds.read_vds(vds_path)
+
+    for chromosome in chromosomes:
+
+        # consider only relevant chromosome
+        vds = hl.vds.filter_chromosomes(vds, keep=chromosome)
```

Comment:
```suggestion
        chrom_vds = hl.vds.filter_chromosomes(vds, keep=chromosome)
```
this will overwrite the vds object, so you need to rename it (otherwise the next loop will find no data remaining on its relevant chromosome)

Reply:
this new chrom-specific name will need to be used in the next few lines as well

Reply:
the lack of available emojis is letting me down -- yes of course i'm dumb, will do this.

Should this be a method / function i call for every chromosome instead though?

Reply:
It depends how complicated what you're doing for each chromosome is.. This is quite short and linear, so you are probably fine as it is